onnxruntime_extensions/pnp/_torchext.py

```diff
@@ -268,4 +268,35 @@ def export(self, *args, **kwargs):
             return super().export(*args, **kwargs)
         else:
             oxml = core_m.export(*args, **kwargs)
-            return ONNXModelUtils.join_models(prefix_m, oxml)
+            model = ONNXModelUtils.join_models(prefix_m, oxml)
+
+            # Rename the input/output node names if the user has provided any substitutions!
+            # Ref: https://github.com/onnx/onnx/issues/2052
+            if (('input_names' in kwargs) or ('output_names' in kwargs)) and \
```

Comment:
One more notes here is that the current code doesn't support updating the node with a sub-graph.

Reply:
Made a note in the comments.